### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by andraskadar**
Set up the tooling with a sample project, fastlane, GH Actions & bundled prebuilt swiftlint

### CODE REVIEW

File: Log/Log/LogApp.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+//  LogApp.swift
+//  Log
+//
+//  Created by Andras Kadar on 18.07.21.
+//
+
+import SwiftUI
+
+@main
+struct LogApp: App {
```
Comment (danieltmbr) [inline Log/Log/LogApp.swift:11]: lol, first I wanted to ask why do we have a log app... :D I thought it was related to the development and not the project itself.
We need to settle for a name I guess.

File: fastlane/Fastfile [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+end
+
+platform :ios do
+  before_all do
+    xcversion(version: Constants::XCODE_VERSION)
+  end
+
+  desc "Lint project"
+  lane :lint_project do |options|
+    reporter = options[:reporter] || 'json'
```
Comment (danieltmbr) [inline fastlane/Fastfile:35]: What's this `reporter`, where was it set up?

File: Log/LogTests/LogTests.swift [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+class LogTests: XCTestCase {
+
+    override func setUpWithError() throws {
+    }
+
+    override func tearDownWithError() throws {
+    }
+
+    func testExample() throws {
+    }
```
Comment (danieltmbr) [inline Log/LogTests/LogTests.swift:20]: Can you lease remove unused code? 😇

**Review by danieltmbr — APPROVED** [review]
Tbh, I'd like to try all the new Xcode 13 beta features and new Swift and iOS 15 improvements, so I'd configure the project with those to be the minimum requirements.

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  lint-and-test:
+    name: Lint and test the project
+    runs-on: macos-10.15
```
[the remark this answers is not in the document]
Comment (andraskadar) [reply .github/workflows/tests.yml:13]: Had to roll back, somehow it was not supported: https://github.com/furry-fiesta/furry-fiesta-ios/pull/3/commits/5a9b105183aa89f17724dd1ae609f6ef8562b66a

File: Log/Log/LogApp.swift (hunk 1 above)
Comment (andraskadar) [reply Log/Log/LogApp.swift:11]: Yes -.-" I think we also need to just delete this project and start a new one when we know the name

File: fastlane/Fastfile (hunk 2 above)
Comment (andraskadar) [reply fastlane/Fastfile:35]: Coming as an optional parameter, maybe need to document it in the comment 🙄

**Pull request closed without merging**